templates/zerver/help/getting-your-organization-started-with-zulip.md

```diff
@@ -122,4 +122,4 @@ users at once, [notifying](/help/pm-mention-alert-notifications) them about a me
 * [Getting started with Zulip](/help/getting-started-with-zulip)
 * [Streams and topics](/help/streams-and-topics)
 * [Moderating open organizations](/help/moderating-open-organizations)
-* [Using Zulip for a class](/help/using-zulip-for-a-class)
+* [Setting up Zulip for your class](/help/setting-up-zulip-for-your-class)
```

Comment:
Hmm, I'm not sure I like the "Your class" framing over "a class" -- since most folks reading this won't have a class.

I could also imagine "Teaching a class on Zulip" being a different label that might be better.

Reply:
Hmm, "Teaching a class on Zulip" doesn't seem right, but I like the your->a change.

Reply:
The insert/delete formatting is misleading, by the way.